src/app/ApplicationTemplate.Presentation/Configuration/NavigationCoreConfiguration.cs

```diff
@@ -24,7 +24,7 @@ public static IServiceCollection AddNavigationCore(this IServiceCollection servi
 	{
 		return services
 			.AddSingleton<IExtendedSplashscreenController, MockExtendedSplashscreenController>()
-			.AddSingleton<ISectionsNavigator>(s => new BlindSectionsNavigator("Login", "Home", "Posts", "Settings"))
+			.AddSingleton<ISectionsNavigator>(s => new BlindSectionsNavigator("Login", "Home", "Posts", "Settings", "KillSwitch"))
```

Comment:
Why do you add a new section? Couldn't you just do the same thing as the forced update?

Reply:
At the beginning, I was planning to just open a new section and then get back the the last section when the kill switch was deactivated, but yeah, with the way things are right now, this should not be necessary, you are right